products/opensuse.md

```diff
@@ -16,8 +16,13 @@ releaseDateColumn: true
 eolColumn: End of Life
 
 identifiers:
+-   cpe: cpe:/o:opensuse:opensuse
 -   cpe: cpe:/o:opensuse:leap
+-   cpe: cpe:/o:opensuse:tumbleweed
+
+-   cpe: cpe:2.3:o:opensuse:opensuse
 -   cpe: cpe:2.3:o:opensuse:leap
+-   cpe: cpe:2.3:o:opensuse:tumbleweed
 
```

Comment:
Tumbleweed should be dropped from here, since it is not represented on the list of releases - it is a rolling, unstable releases so we don't track it as such.

```suggestion
-   cpe: cpe:/o:opensuse:opensuse
-   cpe: cpe:/o:opensuse:leap
-   cpe: cpe:2.3:o:opensuse:opensuse
-   cpe: cpe:2.3:o:opensuse:leap
```